src/shapes/textbox.class.js

```diff
@@ -328,14 +340,25 @@
         words.push([]);
       }
       desiredWidth -= reservedSpace;
-      for (var i = 0; i < words.length; i++) {
+      // measure words
+      var data = words.map(function (word) {
         // if using splitByGrapheme words are already in graphemes.
-        word = splitByGrapheme ? words[i] : fabric.util.string.graphemeSplit(words[i]);
-        wordWidth = this._measureWord(word, lineIndex, offset);
+        word = splitByGrapheme ? word : this.graphemeSplit(word);
+        var width = this.measureWord(word, lineIndex, offset);
+        largestWordWidth = Math.max(width, largestWordWidth);
+        offset += word.length + 1;
```

Comment:
why here is lenght + 1 and before was jsut lenght?

Reply:
I don't really understand how style position is counted.
With spaces or without... Doesn't make sense without. 
Need help

Reply:
The test was wrong
![image](https://user-images.githubusercontent.com/34343793/159441763-287d0784-3cb7-4fc0-811c-f6d98233575b.png)

Reply:
The style of the test textbox object was incomplete.
fixed by ff6b99db